services/identity/service.go

```diff
@@ -110,7 +110,7 @@ func (s *Service) CreateIdentity(si *network.ServerIdentity, ai *CreateIdentity)
 
 	roster := ids.Root.Roster
 	replies, err := manage.PropagateStartAndWait(s.Context, roster,
-		&PropagateIdentity{ids}, 1000, s.Propagate)
+		&PropagateIdentity{ids}, 10000, s.Propagate)
```

Comment:
Instead of `10000` or `1000`, it should be easier to change at a single place using an unexported (or exported?) global variable (or with getter/setter). Even better, we could specify a value to the service (`service.SetTimeout()`) and the service would use the default unexported global variable by default otherwise.